app/build.gradle.kts

```diff
@@ -49,6 +49,7 @@ dependencies {
     implementation(files("libs/borshj-0.0.0.jar"))
 
     Dependencies.baseAndroidLibraries.forEach { implementation(it) }
+    Dependencies.adapterdelegatesLibraries.forEach { implementation(it) }
```

Comment:
AdapterDelegatesLibraries 
and also notify @Android team that you fetch this dependency

Reply:
and why adapterdelegates, and not epoxy for example?

Reply:
because annotationProcessor.
We all agreed adapterdelegates when I present FinanceBlockCell